pkg/utils/release/controller.go

```diff
@@ -394,3 +400,119 @@ func (s *SuiteController) GetSbomPyxisByImageID(pyxisStageURL, imageID string,
 	}
 	return body, nil
 }
+
+// DeleteAllSnapshotEnvironmentBindings removes all CRs SnapshotEnvronmentBindings in cluster.
+func (s *SuiteController) DeleteAllSnapshotEnvironmentBindings() error {
```

Comment:
I guess `SnapshotEnvironmentBinding` release-service specific CRD - would it make sense to move the method to common controller?

Reply:
I don't have any problem moving it to common, yet it's a release-service specific and this why I thought to create it in release/controller .